README.md

```diff
@@ -11,7 +11,7 @@ The following code snippet shows the most basic features of `DDSketch`:
 ```java
 // Creating an initially empty sketch, with low memory footprint
 double relativeAccuracy = 0.01;
-DDSketch sketch = DDSketch.memoryOptimal(relativeAccuracy);
+DDSketch sketch = DDSketches.unboundedDense(relativeAccuracy);
```

Comment:
can we create a constructor so a user can just do:
`DDSketch sketch = DDSketch(relativeAccuracy)`
and make the default be a collapsing lowest dense (with some high bin limit)?

Reply:
I'd like to avoid favoring any store for now in `DDSketch`, or maybe not until we have one implemented that unequivocally fits most if not all use cases more than any other.

Reply:
it's fine if we favor one now and switch it later, right?

Reply:
It could still cause backward compatibility issues (e.g., if users rely on the behavior of the store or if they cast it to a specific implementation of the store), which is why I'd like to avoid it. I added a reference to `DDSketches` from the constructors in 477f8ea87f9920268af5aed83f45c0fa571cb50d, as a way to further highlight the existence of those factory methods.

Reply:
Can we update the initial README snippet to use the basic constructor instead?